drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/referral_household_member.rb

```diff
@@ -12,7 +12,7 @@ class ReferralHouseholdMember < ::HmisExternalApis::HmisExternalApisBase
     belongs_to :client, class_name: 'Hmis::Hud::Client'
 
     enum(
-      relationship_to_hoh: ::HudUtility.hud_list_map_as_enumerable(:relationship_to_ho_h_map),
+      relationship_to_hoh: ::HudUtility.hud_list_map_as_enumerable(:relationships_to_hoh),
```

Comment:
following the same thread of `_options` suffix, that would make pluralization way cleaner.  It'd be `relationship_to_hoh_options`

Reply:
The overrides are there specifically to match the function names that are currently in use in HudUtility. HudUtility has `relationships_to_hoh`, for example, so I added an override to name the code-gen'd replacement the same way. I think thats what we agreed when we chatted last week. It lets us avoid doing a bunch of find-and-replaces for all the new fn names.

Reply:
Yep, that's what we agreed on.  Also, _maybe_ some aliases?

Reply:
I was having trouble getting aliases to work for some reason, I couldn't alias methods that were included via the concern. What are you thinking in terms of aliases?

Reply:
I don't really think they are necessary, just trying to cover extra bases.  Let's leave it as is